src/core/thread/mle.hpp

```diff
@@ -1247,7 +1253,9 @@ class Mle : public InstanceLocator
     otError SendDataRequest(const Ip6::Address &aDestination,
                             const uint8_t *     aTlvs,
                             uint8_t             aTlvsLength,
-                            uint16_t            aDelay);
+                            uint16_t            aDelay,
+                            const uint8_t *     aExtraTlvs,
+                            uint8_t             aExtraTlvsLength);
```

Comment:
Declare default values for the new parameters so that no need to change previous usages?

```suggestion
                            const uint8_t *     aExtraTlvs = nullptr,
                            uint8_t             aExtraTlvsLength = 0);
```